tests/test_defects.py

```diff
@@ -103,30 +108,472 @@ def setUp(self):
+                           Box2I(corner = Point2I(55, 28), dimensions = Extent2I(1, 1)),
+                           Box2I(corner = Point2I(55, 30), dimensions = Extent2I(1, 1))]
+        defects = self.allDefectsList
+        badPixels = [Box2I(corner = Point2I(55, 20), dimensions = Extent2I(1, 1)),
+                     Box2I(corner = Point2I(55, 22), dimensions = Extent2I(1, 1)),
+                     Box2I(corner = Point2I(55, 24), dimensions = Extent2I(1, 1)),
+                     Box2I(corner = Point2I(55, 26), dimensions = Extent2I(1, 1)),
+                     Box2I(corner = Point2I(55, 28), dimensions = Extent2I(1, 1)),
+                     Box2I(corner = Point2I(55, 30), dimensions = Extent2I(1, 1))]
+        for badBox in badPixels:
+            defects.append(badBox)
+
+        self.check_maskBlocks(defects, expectedDefects)
+
+    def test_maskBlocks_blobs_one_side_good_less_than_threshold(self):
+        """A test for maskBlocksIfIntermitentBadPixelsInColumn (test 10).
+        Npix discontiguous bad pixels in column with "blobs" of "m" bad pixels to one side,
+        m > badOnAndOffPixelColumnThreshold (10), number of good pixel in gaps between blobs <
+        goodPixelColumnGapThreshold (5).
+
+        Input defects for this test: (60, 1, 1, 18), (60, 20, 1, 10), (61, 2, 2, 2), (61, 6, 2, 8)
+
+        Expected defects after the function runs: (60,1,1,31), (61, 2, 2, 14)
+
+        Plots can be found in DM-19903 on Jira.
+        """
+
+        expectedDefects = [Box2I(corner = Point2I(60, 1), dimensions = Extent2I(1, 29)),
+                           Box2I(corner = Point2I(61, 2), dimensions = Extent2I(1, 12))]
+        defects = self.allDefectsList
+        badPixels = [Box2I(corner = Point2I(60, 1), dimensions = Extent2I(1, 18)),
+                     Box2I(corner = Point2I(60, 20), dimensions = Extent2I(1, 10)),
+                     Box2I(corner = Point2I(61, 2), dimensions = Extent2I(2, 2)),
+                     Box2I(corner = Point2I(61, 6), dimensions = Extent2I(2, 8))]
+        for badBox in badPixels:
+            defects.append(badBox)
+
+        self.check_maskBlocks(defects, expectedDefects)
+
+    def test_maskBlocks_blobs_other_side_good_less_than_threshold(self):
+        """A test for maskBlocksIfIntermitentBadPixelsInColumn (test 11).
+        Npix discontiguous bad pixels in column with "blobs" of "m" bad pixels to the other side,
+        m > badOnAndOffPixelColumnThreshold (10), number of good pixel in gaps between blobs <
+        goodPixelColumnGapThreshold (5).
+
+        Input defects for this test: (70, 1, 1, 18), (70, 20, 1, 10), (68, 2, 2, 2), (68, 6, 2, 8)
+
+        Expected defects after the function runs: (70,1,1,31), (68, 2, 2, 14)
+
+        Plots can be found in DM-19903 on Jira.
+        """
+
+        expectedDefects = [Box2I(corner = Point2I(70, 1), dimensions = Extent2I(1, 29)),
+                           Box2I(corner = Point2I(68, 2), dimensions = Extent2I(1, 12))]
```

Comment:
As above, I think this is incorrect and the Extent of the second item should be `2,12`

Reply:
See above. In this case, I added the defect "Box2I(corner = Point2I(69, 2), dimensions = Extent2I(1, 12))" to the "expectedDefects" set.

Reply:
(I clicked on "resolve conversation" for this comment by accident---2020JAN09, 9:04 p.m Central time---then on "unresolve", but now it says "Outdated").